system/greenhouse-ccloud/templates/thanos-global.yaml

```
@@ -0,0 +1,78 @@
+{{- if and .Values.thanos.enabled .Values.thanos.globalClusters -}}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: thanos-{{ .Chart.Name }}-tls
+  namespace: {{ .Chart.Name }}
+type: Opaque
+data:
+  ca.crt: {{ required ".Values.thanos.tls.caCert missing" .Values.thanos.tls.caCert | b64enc | quote }} 
+  tls.crt: {{ required ".Values.thanos.tls.tlsCert missing" .Values.thanos.tls.tlsCert | b64enc | quote }} 
+  tls.key: {{ required ".Values.thanos.tls.tlsKey missing" .Values.thanos.tls.tlsKey | b64enc | quote }}
+---
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginPreset
+metadata:
+  name: thanos-global
+  namespace: {{ .Chart.Name }}
+spec:
+  clusterSelector:
+    matchExpressions:
+    - key: greenhouse.sap/pluginpreset
+      operator: In
+      values:
+      - "true"
+{{- with .Values.thanos.globalClusters }}
+    - key: cluster
+      operator: In
+      values:
+        {{- range $cluster := $.Values.thanos.globalClusters }}
+        - {{ $cluster.name }}
+        {{- end }}
+  clusterOptionOverrides:
+  {{ include "thanos.storeEndpoints" $ | nindent 4 }}
+{{- end }}  
+  plugin:
+    disabled: false
+    displayName: Thanos global
+    pluginDefinition: thanos
+    releaseNamespace: kube-monitoring
+    optionValues:
+    {{- if and .Values.global.quayIoMirror .Values.global.ghcrIoMirror .Values.global.dockerHubMirror }}
+    - name: thanos.image.repository
+      value: {{ printf "%s/thanos/thanos" .Values.global.quayIoMirror }}
+    - name: thanos.initChownData.image.registry
+      value: {{ .Values.global.dockerHubMirror }}
+    - name: testFramework.image.registry
+      value: {{ .Values.global.ghcrIoMirror }}
+    {{- end }}
+    - name: thanos.query.autoDownsampling
+      value: false
```

Any particular reason for this?

---

No. This is a mistake 
```suggestion
    - name: thanos.query.autoDownsampling
      value: true
```